department-of-veterans-affairs/caseflow
# Add supplemental claim review

Resolves #16317 

### Description
Use form generator on supplemental claim page

### Acceptance Criteria
- everything works the same on the supplemental claim review page

### Testing Plan
1. Go to intake
2. Start a supplemental claim
3. ensure all validation works and that you can submit
